src/ui/simulator/cmake/windows-inspector.cmake

```diff
@@ -25,6 +25,14 @@ SET(SRC_WINDOW_INSPECTOR
 
 add_library(antares-ui-windows-inspector	${SRC_WINDOW_INSPECTOR})
 
-target_link_libraries(antares-ui-windows-inspector PUBLIC  ${wxWidgets_LIBRARIES})
-target_link_libraries(antares-ui-windows-inspector PRIVATE antares-ui-common)
+target_link_libraries(antares-ui-windows-inspector
+		PUBLIC
+		${wxWidgets_LIBRARIES})
+target_link_libraries(antares-ui-windows-inspector
+		PRIVATE
+		antares-ui-common
+		Antares::resources
+		Antares::locator

```

Comment:
Why would we need the **locator** in the **inspector** ?
It does not seem to make sens.
Same thing for other newly linked library.

Reply:
inspector use ui/simulator/study.h which include antares/locator/locator.h
antares/locator/locator.h need to be known by inspector

Infortunatly UI libraries are not compliant with PUBLIC/PRIVATE cmake proper usage. To put the dependency at the right place we would need to clean up UI libraries which is not worth it